charts/incubator/blocky/docs/installation-notes.md

```diff
@@ -0,0 +1,13 @@
+# Installation notes
+
+Blocky supports both single config file and multiple (split config files).
+If you want to use multiple config files, set `useConfigDirectory` to true.
+
+This will copy a basic config file (named `tc-config.yaml`) with redis and prometheus, and a default upstream
+DNS of `1.1.1.1`. You can edit this file afterwards. If you ever need to recreate that file, simple delete it
+and restart the Chart.
+
+Scale users can change the `config` persistence to `HostPath (Simple)` and mount it to a directory on host.
+There you can add you config files.
```

Comment:
Honestly, I don't want this.

The unique sellingpoint of blocky is that it can be setup compeletely stateless using configmaps.